bricks/vs/src/code-editor/utils/celCompletionProvider.ts

```
@@ -0,0 +1,220 @@
+import * as monaco from "monaco-editor/esm/vs/editor/editor.api.js";
+import { hasOwnProperty } from "@next-core/utils/general";
```

Biome

[error] Do not shadow the global "hasOwnProperty" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)